### PR TITLE
volume annotation syntax error

### PR DESCRIPTION
**Pull request by AlexMioMio**
correct annotation syntax error

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35604)

### CODE REVIEW

**Comment by AlexMioMio** [comment]
@googlebot I signed it!

**Comment by AlexMioMio** [comment]
@googlebot I signed it!

**Comment by saad-ali** [comment]
I think the original comment makes sense as is. It describes *how*. Removing that part removes information. @vishh who was the original author for final say.

**Comment by AlexMioMio** [comment]
@k8s-bot gci gce e2e test this

File: pkg/volume/metrics_statfs.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 var _ MetricsProvider = &metricsStatFS{}
 
 // metricsStatFS represents a MetricsProvider that calculates the used and available
-// Volume space by stat'ing and gathering filesystem info for the Volume path.
```
Comment (vishh) [inline pkg/volume/metrics_statfs.go]: stat'ing is a keyword here which describes what the method does.

**Comment by vishh** [comment]
Why open this PR?

File: pkg/volume/metrics_statfs.go (hunk 1 above)
Comment (AlexMioMio) [inline pkg/volume/metrics_statfs.go]: yeah,i will be more careful next time,thx

**Pull request closed without merging**